crates/sdk/libs/near/src/query.rs

```diff
@@ -93,7 +35,7 @@ pub enum QueryResponseKind {
 
 impl RpcMethod for RpcQueryRequest {
     type Response = RpcQueryResponse;
-    type Error = RpcQueryError;
```

Comment:
seems you were on the right path, what happened?

Reply:
Yep. But I thought I was overengeenering.

Reply:
Ah, it's just the right amount.